shakenfist/constants.py

```diff
@@ -0,0 +1,12 @@
+# Possible object states
+
+STATE_INITIAL = 'initial'
```

Comment:
These should probably live with the Instance object.

They could also have a prefix (eg. `instance-`) to ensure the correct states are assigned to the correct object.

Reply:
So a bunch of these states are used in other object types... This is to guard against typos, not an object using a state it shouldn't (the state transition code guards against that).

We should talk more about this one.

Reply:
Done.